contrib/grinder/properties/grinder.properties.qe01

```diff
@@ -17,7 +17,7 @@ grinder.bf.metrics_per_tenant=210
 grinder.bf.batch_size=150
 grinder.bf.ingest_concurrency=25
 grinder.bf.annotations_concurrency=5
-grinder.bf.num_nodes=1
+grinder.bf.num_nodes=2
```

Comment:
What does this number of nodes refer to?  We only have one shim00 server in qe01/qe02 environments.